### PR TITLE
🦤 Fix Qualcomm configure doc/comments

### PR DESCRIPTION
**Pull request by trajepl**
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by guotuofeng — DISMISSED** [review]

File: examples/inception/README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ Unzip the file and set the unzipped directory path as environment variable `SNPE
 ### Configure SNPE
 ```sh
 # in general, python 3.8 is recommended
-python -m olive.platform_sdk.qualcomm.snpe.configure --py_version 3.8
+python -m olive.platform_sdk.qualcomm.configure --py_version 3.8
```
Comment (jambayk) [inline examples/inception/README.md:20]: This does not have the sdk provided but it is required https://github.com/microsoft/Olive/blob/main/olive/platform_sdk/qualcomm/configure.py#L97

**Review by jambayk — CHANGES_REQUESTED** [review]
missing sdk

File: examples/inception/README.md (hunk 1 above)
Comment (jambayk) [reply examples/inception/README.md:20]: can you update here too? https://github.com/microsoft/Olive/blob/main/examples/vgg/README.md

Comment (trajepl) [reply examples/inception/README.md]: Thanks. Done.

**Review by jambayk — APPROVED** [review]

**Pull request merged**